src/test/java/com/dhex/shipping/services/ShippingServiceTest.java

```diff
@@ -1,19 +1,89 @@
 package com.dhex.shipping.services;
 
+import com.dhex.shipping.exceptions.NotValidShippingStatusException;
+import com.dhex.shipping.exceptions.ShippingNotFoundException;
 import com.dhex.shipping.model.ShippingRequest;
+import com.dhex.shipping.model.ShippingStatus;
+import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
 
 import static org.hamcrest.core.Is.is;
 import static org.hamcrest.core.IsNull.notNullValue;
 import static org.junit.Assert.*;
 
 public class ShippingServiceTest {
+
+    ShippingRequest shippingRequest;
+
+    private ShippingService shippingService;
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Before
+    public void setUp() throws Exception {
```

Comment:
If we are not expecting any exception, we can omit the throws part in the signature. If we were expecting exceptions, we should declare them explicitly ([don't catch generic exceptions](http://www.javaworld.com/article/2073800/testing-debugging/beware-the-dangers-of-generic-exceptions.html))